apps/docs/components/Header.tsx

```diff
@@ -69,7 +69,7 @@ export function SectionLinks({ sectionId }: { sectionId?: string | null }) {
 				data-active={sectionId === 'reference'}
 				className="layout_header__section"
 			>
-				Reference
+				API
```

Comment:
this should still say 'Reference' i think - it makes it consistent with other doc sites, and besides, "API" can feel overloaded/vague a bit.